.env.dc

```diff
@@ -1,6 +1,6 @@
 ES_URL=http://elasticsearch_dev:9200
 NEO4J_URL=bolt://neo4j_dev:7687
 FILE_SERVER=http://localhost:8080
-UPLOAD_FOLDER=uploads/
+UPLOAD_FOLDER=uploads
```

Comment:
It is important to remove the dangling slash here because docker-compose would not mount the volume properly with it. 
The change does not break the normal upload workflows in the code!